code/datums/round_modifiers/_round_modifier.dm

```diff
@@ -0,0 +1,66 @@
+/datum/round_modifier
+	var/name = null
+	var/desc = null
+	var/permament = FALSE
+
+/datum/round_modifier/proc/on_apply()
+	return
+
+/datum/round_modifier/proc/on_remove()
+	return
+
+/datum/round_modifier/proc/announce_enabling()
+	priority_announce("A bug has occured with the universe. Please notify a Centcom employee immediately.")
+
+/datum/round_modifier/proc/announce_disabling()
```

Comment:
Unless I'm blind these two procs aren't called anywhere, intentional?

Reply:
Yeah, they're not. In a hypothetical future, when enabling/disabling, Centcom messages could be sent to tell people IC about the various rules. But that's not implemented right now.

Reply:
You've got two modifiers with announcements defined, implement it =P